lib/lhm/throttler/slave_lag.rb

```diff
@@ -15,7 +15,7 @@ def initialize(options = {})
         @timeout_seconds = INITIAL_TIMEOUT
         @stride = options[:stride] || DEFAULT_STRIDE
         @allowed_lag = options[:allowed_lag] || DEFAULT_MAX_ALLOWED_LAG
-        @slave_connections = {}
+        @get_current_shard = options[:get_current_shard]
```

Comment:
why not just call this get config or something like that, the shard part is orthogonal?